handlers/k8sconfig_handler.go

```diff
@@ -234,7 +234,12 @@ func (h *Handler) checkIfK8SConfigExistsOrElseLoadFromDiskOrK8S(req *http.Reques
 		}
 		prefObj.K8SConfig = kc
 		ctxID := "0" // To be replaced after multi-context support
-		go registerK8sComponents(h.log, prefObj.K8SConfig.Config, ctxID)
+		go func(l logger.Handler, config []byte, ctx string) {
+			err := registerK8sComponents(h.log, prefObj.K8SConfig.Config, ctxID)
+			if err != nil {
+				logrus.Error(err)
```

Comment:
switch to MeshKit errors, please.

Reply:
the error returned by registerK8sComponents are meshkit errors only